tests/test_raster.py

```diff
@@ -359,3 +359,38 @@ def test_zonal_stats():
         da.raster.zonal_stats(gdf, "asdf")
     with pytest.raises(IndexError, match="All geometries outside raster domain"):
         da.raster.zonal_stats(gdf.iloc[1:2], "mean")
+
+
+@pytest.fixture
+def dummy():
+    rast = raster.full_from_transform(
+        transform=[0.004166666666666666, 0.0, 0.0, 0.0, -0.004166666666666667, 0.0],
+        shape=(21600, 21600),
```

Comment:
Would it be possible to test with smaller data?

Reply:
Yes.